README.md

```diff
@@ -67,6 +67,9 @@ Trunk uses a source HTML file to drive all asset building and bundling. Trunk al
   <head>
     <link data-trunk rel="scss" href="path/to/index.scss"/>
   </head>
+  <body>
+    <div id="app"></div>
+  </body>
```

Comment:
Given that this is a bit framework dependent, I'm going to remove this readme change once I merge. I definitely want to keep the cognitive load as minimal as possible on new users and such.